Makefile

```diff
@@ -6,10 +6,23 @@ export PATH := $(PATH):$(GOPATH)/bin
 INSTALL := /usr/bin/install
 DBDIR := /var/run/redis/sonic-db/
 GO ?= /usr/local/go/bin/go
-TOP_DIR := $(abspath ..)
-MGMT_COMMON_DIR := $(TOP_DIR)/sonic-mgmt-common
+TOPDIR := .
+MGMT_COMMON_DIR := $(TOPDIR)/../sonic-mgmt-common
```

Comment:
MGMT_COMMON_DIR must be an absolute path.. It is used to set env variables like CVL_SCHEMA_PATH etc, used during gotests

Reply:
Thanks @sachinholla . corrected..